pkg/build/builder/cmd/scmauth/util.go

```diff
@@ -40,7 +40,7 @@ func ensureGitConfigIncludes(path string, context SCMAuthContext) error {
 		return createGitConfig(path, context)
 	}
 
-	lines, err := file.ReadLines(gitconfig)
+	lines, err := builder.ReadLines(gitconfig)
```

Comment:
This method also seems like a library-go candidate

Reply:
will do, but the fact that i was able to move it would imply no one else is using it today.

Reply:
@deads2k veto'd putting this in library-go so i'm leaving it here.

Reply:
> This method also seems like a library-go candidate



> @deads2k veto'd putting this in library-go so i'm leaving it here.

This is the only consumer and the method is questionable complexity for library-go.